lib/prisma/alumni.ts

```diff
@@ -0,0 +1,61 @@
+import { prisma } from '@/lib/prisma/prisma';
+import { AlumCreateRequest } from 'pages/types/apiRequests';
+import { isNull } from 'lodash';
+import { exclude } from './helpers';
```

Comment:
could you try omit of lodash

Reply:
I followed guide from [prisma](https://www.prisma.io/docs/concepts/components/prisma-client/excluding-fields#excluding-the-password-field). It LGTM

Reply:
https://lodash.com/docs/4.17.15#omit
It's totally the same :v